BeVolt/BSP/Simulator/DataGeneration/Contactor.py

```diff
@@ -0,0 +1,16 @@
+import csv
+
+#This module will read from the csv file to determine the status of the Contactor
+
+#file path to data
+file = "Data/Contactor.csv"
+
+#Reads file and returns 1 or 0, 1 is on, 0 is off
+def Read_Contactor():
+    with open(file, 'r') as csvfile:
+        csvreader = csv.reader(csvfile)
+        return next(csvreader)
+        #print(next(csvreader)): used this to test function
```

Comment:
Go to the issues ticket title **Simulator: Locking files during read and writes** and add the necessary code to safely access the files.